internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,21 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+// fillClusterIDFromMigrationReq fills the clusterID for the passed in monitors.
```

Comment:
Move this function to util, as it not rbd specific ?

Reply:
the migration is only applicable for rbd now, so its fine to keep it here.